doc/manual/expressions/builtins.xml

```diff
@@ -1023,10 +1023,18 @@ in foo</programlisting>
 
     <listitem><para>Convert the expression
     <replaceable>e</replaceable> to a string.
-    <replaceable>e</replaceable> can be a string (in which case
-    <function>toString</function> is a no-op), a path (e.g.,
-    <literal>toString /foo/bar</literal> yields
-    <literal>"/foo/bar"</literal> or a set containing <literal>{ __toString = self: ...; }</literal>.</para></listitem>
+    <replaceable>e</replaceable> can be:
+    <itemizedlist>
+        <listitem>a string (in which case <function>toString</function> is the identity)</listitem>
```

Comment:
`is the identity` is a regression here, not everyone knows what it means. Let's keep `no-op`.